src/.nuget/Microsoft.Build.Mono.NoSymbols.nuspec

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>$id$</id>
+    <version>14.1.0.0-prerelease</version>
+    <title>Microsoft Build Engine</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>https://github.com/Microsoft/msbuild/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/msbuild</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>The Microsoft Build Engine is a platform for building applications.</description>
+    <summary>The Microsoft Build Engine, which is also known as MSBuild, provides an XML schema for a project file that controls how the build platform processes and builds software. Visual Studio uses MSBuild, but MSBuild does not depend on Visual Studio. By invoking msbuild.exe on your project or solution file, you can orchestrate and build products in environments where Visual Studio isn't installed.</summary>
+    <releaseNotes>Initial release</releaseNotes>
+    <copyright>Copyright &#169; Microsoft Corporation</copyright>
+    <tags>Build MSBuild</tags>
+    <references>
+      <reference file="Microsoft.Build.dll" />
+      <reference file="Microsoft.Build.Framework.dll" />
+      <reference file="Microsoft.Build.Tasks.Core.dll" />
+      <reference file="Microsoft.Build.Utilities.Core.dll" />
+    </references>
+    <dependencies>
+      <dependency id="Microsoft.Tpl.Dataflow" version="4.5.24" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="$bin$/Microsoft.Build.dll" target="lib" />
```

Comment:
Any reason for target="lib"? Seems like the convention is target="tools" (e.g used by FAKE).